libcloud/loadbalancer/drivers/dimensiondata.py

```diff
@@ -559,8 +560,8 @@ def ex_create_virtual_listener(self,
                                    network_domain_id,
                                    name,
                                    ex_description,
-                                   port,
                                    pool,
+                                   port=None,
```

Comment:
No, it won't. We're just setting the value to None here if port is not specified.